src/containerapp/azext_containerapp/commands.py

```diff
@@ -34,11 +27,6 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_replicas')
```

Comment:
why do we have replica left?

Reply:
replica count is a new command, in that PR changed a little exception logic for other replica commands too.